nginx.conf

```diff
@@ -0,0 +1,13 @@
+events{}
+http {
+    include /etc/nginx/mime.types;
+    server {
+        listen 80;
+        server_name localhost;
+        root /usr/share/nginx/html;
+        index index.html;
+        location / {
+            try_files $uri $uri/ /index.html;
```

Comment:
that is unsafe location `try_files` as it allows you to download files by name, for example nginx.conf - which is a security hole.